corekit/src/main/java/com/midtrans/sdk/corekit/base/callback/MidtransCallback.java

```diff
@@ -1,6 +1,6 @@
 package com.midtrans.sdk.corekit.base.callback;
 
-public interface MidtransCallback<T> {
+public interface MidtransCallback<T> extends HttpRequestCallback{
```

Comment:
I think for now `HttpRequestCallback`not needed, because we can wrap all error or throwable into `onFailed()`, in previous version of SDK there are 3 callback and user must implement it, so it more difficult for user to handling the error because there are 2 error callback. I think it more better if we define the throwable and wrap it in `onFailed()`, so user can handle all of error in `onFailed()` callback. How about it?

Reply:
updated